pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
Cool, seems like my intellij project had some broken paths from being moved to a different place on my filesystem, I think I'm all sorted now.